api/src/routes/certificate.ts

```diff
@@ -0,0 +1,389 @@
+          }
+        });
+
+        const updatedIsCertMap = getUserIsCertMap(removeNulls(updatedUser));
+
+        const certMap = Object.entries(updatedIsCertMap);
+        // TODO(POST-MVP): Consider sending email based on `user.isEmailVerified` as well
+        const shouldSendCertifiedEmailToCamper =
+          isEmail(updatedUser.email) &&
+          currentCertifications.every(cert => {
+            const certIsClaimed = certMap.some(
+              ([name, isClaimed]) => name === certSlugTypeMap[cert] && isClaimed
+            );
+            return certIsClaimed;
+          });
+
+        if (shouldSendCertifiedEmailToCamper) {
+          const notifyUser = {
+            to: updatedUser.email,
+            from: 'quincy@freecodecamp.org',
+            subject:
+              'Congratulations on completing all of the freeCodeCamp certifications!',
+            text: renderCertifiedEmail({
+              username: updatedUser.username,
+              // Safety: `user.name` is required to exist earlier. TODO: Assert
+              name: updatedUser.name as string
+            })
+          };
+
+          // Failed email should not prevent successful response.
+          try {
+            // TODO(POST-MVP): Ensure Camper knows they **have** claimed the cert, but the email failed to send.
+            await fastify.sendEmail(notifyUser);
+          } catch (e) {
+            fastify.log.error(e);
+            // TODO: Log to Sentry
+          }
+        }
+
+        void reply.code(200);
+        return {
+          response: {
+            type: 'success',
+            message: 'flash.cert-claim-success',
+            variables: {
+              username: updatedUser.username,
+              name: certName
+            }
+          },
+          isCertMap: updatedIsCertMap,
+          completedChallenges: updatedUser.completedChallenges
+        } as const;
+      } catch (e) {
+        fastify.log.error(e);
+        void reply.code(500);
+        throw {
+          type: 'danger',
+          // message: 'Oops! Something went wrong. Please try again in a moment or contact
+          message: 'flash.went-wrong'
+        } as const;
+      }
+    }
+  );
+
+  done();
+};
+
+// TODO: Current api is a bit LB specific. Look into templating.
+function renderCertifiedEmail({
+  username,
+  name
+}: {
+  username: string;
+  name: string;
+}) {
+  const certifiedEmailTemplate = `Hi ${name || username},
+
+Congratulations on completing all of the freeCodeCamp certifications!
+
+All of your certifications are now live at at: https://www.freecodecamp.org/${username}
+
+Please tell me a bit more about you and your near-term goals.
+
+Are you interested in contributing to our open source projects used by nonprofits?
+
+Also, check out https://contribute.freecodecamp.org/ for some fun and convenient ways you can contribute to the community.
+
+Happy coding,
+
+- Quincy Larson, teacher at freeCodeCamp
+`;
+  return certifiedEmailTemplate;
+}
```

Comment:
To discuss:

`api-server/` uses Loopback to template this. Do we need/want to do that in the new api?

I do not see the need, and do not want to add the complexity.

Reply:
Discussed in meeting: As long as functionality is the same, this is fine as is.